spec/models/range_request_spec.rb

```diff
@@ -116,23 +137,105 @@
     expect(info[:is_multipart]).to be_truthy
 
     expect(info[:range_start_bytes][0]).to eq(0)
-    expect(info[:range_end_bytes][0]).to eq(range_request.max_range_size)
```

Comment:
Oh, I see,  I compared the range start/end value to the length - which is off by one, since the value is a zero-based, inclusive index. Makes sense.

Just to confirm: if `max_range_size` is 10, and `range_end_bytes` is as high as it can be, then `range_end_bytes` would be 9? So need to subtract one from the max length of 10.